reporter/src/funTest/assets/static-html-reporter-test-input.yml

```diff
@@ -566,8 +584,8 @@ scanner:
           start_time: "1970-01-01T00:00:00Z"
           end_time: "1970-01-01T00:00:00Z"
           package_verification_code: "0000000000000000000000000000000000000000"
-          licenses: [ ]
-          copyrights: [ ]
+          licenses: []
+          copyrights: []
```

Comment:
Nit: This seems to be a pure formatting change unrelated to the model changes, but I'm ok with keeping it as part of this commit.